core/src/main/java/com/linecorp/armeria/common/AbstractContextAwareBlockingTaskExecutor.java

```diff
@@ -15,14 +15,12 @@
  */
 package com.linecorp.armeria.common;
 
-final class RequestContextUtil {
```

Comment:
This class has been merged to `com.linecorp.armeria.internal.common.RequestContextUtil` to avoid confusion.